### ISSUE TITLE
Netlify: Request must be smaller than 69905067 bytes for the CreateFunction operation

### ISSUE THREAD

**Issue by adyz** [remix-run/remix#643]
### Which Remix packages are impacted?

- [ ] `remix` (Remix core)
- [ ] `create-remix`
- [ ] `@remix-run/architect`
- [ ] `@remix-run/cloudflare-workers`
- [ ] `@remix-run/dev`
- [ ] `@remix-run/express`
- [X] `@remix-run/netlify`
- [ ] `@remix-run/node`
- [ ] `@remix-run/react`
- [ ] `@remix-run/serve`
- [ ] `@remix-run/server-runtime`
- [ ] `@remix-run/vercel`

### What version of Remix are you using?

1.0.4

### Steps to Reproduce

Create a new Remix project and install/setup netlify cli

`npx create-remix@latest test-deploy`

Add one large dependency (tried just with `react-spring`)

`npm i react-spring`

import and use the dependency

`import { animated } from 'react-spring';`

```jsx
const Boop = ({ children}: any) => {
    return (
      <animated.span>
        {children}
      </animated.span>
    );
  };
````

`npm run build` works fine
`npm run dev:netlify` works file
`netlify deploy` **Request must be smaller than 69905067 bytes for the CreateFunction operation**

<img width="1231" alt="Screenshot 2021-11-25 at 18 15 55" src="https://user-images.githubusercontent.com/611174/143474749-f21ef187-6cc5-4064-a6ff-955816945a58.png">

### Expected Behavior

I expect to be able to run that on netlify without any deploy problem.

I have no ideea where to go from here. I've searched online and it seems like a serverless issue, but not specific to netlify

### Actual Behavior

It hangs at "Request must be smaller than 69905067 bytes for the CreateFunction operation"

**Comment by adyz** [comment]
I've created test repo: [https://github.com/adyz/remix-test](https://github.com/adyz/remix-test)

If you go to the pull requests tab I've created two pull request to trigger the netlify deploy

One PR deploys just fine and it's based on the default create-remix@latest template.
The second one has one new dependency (react-spring) and it's imported in the index route. This one fails to deploy to netify.

Any help is greatly appreciated.

Remix looks so nice. I so excited about remix. It make development interesting for me again. 👯
